### PR TITLE
[FSDE-15311] Migrate Toggle switch component to new vue documentation

### PR DESCRIPTION
**Pull request by soujanreddy**
@jllr We need a ticket number for this component.

### CODE REVIEW

**Review by josecarlosgonzalezv — APPROVED** [review]

File: src/documentation/pages/components/forms/toggle-switch/examples/_inline-lumen-centurylink.vue [hunk 1]
```diff
@@ -0,0 +1,75 @@
+<template lang="pug">
+  <ComponentExample title="Inline" id="inline-portal" :tabs="exampleTabs">
```
Comment (evansito) [inline src/documentation/pages/components/forms/toggle-switch/examples/_inline-lumen-centurylink.vue:2]: this should be inline-lumen-centurylink

File: src/documentation/pages/components/forms/toggle-switch/examples/_portal/_base-portal.vue [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      <span class="chi-switch__thumb"></span>
+    </span>
+    <span class="chi-switch__label">Label</span>
+  </label>
+</div>`,
+      },
+    };
+  },
+})
+export default class BasePortal extends Vue {}
```
Comment (evansito) [inline src/documentation/pages/components/forms/toggle-switch/examples/_portal/_base-portal.vue:46]: this file doesn't need to be named base-portal, as it's inside a portal folder, you can name it just _base

File: src/documentation/pages/components/forms/toggle-switch/examples/_portal/_checked-portal.vue [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+      <span class="chi-switch__thumb"></span>
+    </span>
+    <span class="chi-switch__label">Label</span>
+  </label>
+</div>`,
+      },
+    };
+  },
+})
+export default class CheckedPortal extends Vue {}
```
Comment (evansito) [inline src/documentation/pages/components/forms/toggle-switch/examples/_portal/_checked-portal.vue:49]: Same here and in other similar files within portal/

**Review by evansito — CHANGES_REQUESTED** [review]

**Review by evansito — CHANGES_REQUESTED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-327/6/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://21c6509472cb5c27c2cfe2c946ed6da352d458e8-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-327-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Review by evansito — APPROVED** [review]

File: src/documentation/pages/components/forms/toggle-switch/examples/_portal/_sizes.vue [hunk 4]
```diff
@@ -0,0 +1,75 @@
+<template lang="pug">
+  <ComponentExample title="Sizes" id="sizes-portal" :tabs="exampleTabs">
+    .p-text(slot="example-description" class="-mb--3 -text")
+      | Toggle switch supports different sizes: <code>xs</code>, <code>sm</code>.
+      | The default size is <code>sm</code>.
```
Comment (jllr) [inline src/documentation/pages/components/forms/toggle-switch/examples/_portal/_sizes.vue:5]:
```suggestion
      | The default size is <code>xs</code>.
```

**Review by jllr — CHANGES_REQUESTED** [review]

**Review by jllr — APPROVED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-327/7/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://89e4f31b877ca18edfdc46d75c8484f2415fc45f-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-327-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://11b57e5022eae091efe4096979f55975508f66e6-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-327-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-327/9/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://b989cf354530ddb6718670ee28b2ecef8b03fafc-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-327-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Pull request merged**